py/src/braintrust/logger.py

```diff
@@ -2207,17 +2226,32 @@ def log_internal(self, event=None, internal_data=None):
             **sanitized_and_internal_data,
             **{IS_MERGE_FIELD: self._is_merge},
         )
-        serialized_partial_record = _check_json_serializable(partial_record)
-        partial_record = json.loads(serialized_partial_record)
-        if "metrics" in partial_record and "end" in partial_record["metrics"]:
-            self._logged_end_time = partial_record["metrics"]["end"]
 
-        if len(partial_record.get("tags", [])) > 0 and self.span_parents:
+        serializable_partial_record = {}
```

Comment:
@manugoyal

Reply:
As long as it mirrors the JS implementation, this seems fine / same questions/comments apply.